h/schemas/forms/admin/group.py

```diff
@@ -153,6 +157,7 @@ def __init__(self, *args):
         title=_("Organization"),
         description=_("Organization which this group belongs to"),
         widget=group_organization_select_widget,
+        missing=None,
```

Comment:
Allow `organization` to be missing from create/edit group admin form submissions (this is what happens if the user chooses `-- None --`)